tests/compare_INIT.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import subprocess
+import sys
+import os.path
+
+from ert.ecl import EclFile
+from ert.test import TestAreaContext
+
+
+def compare_files( flow_file , ref_file):
+    flow = EclFile( flow_file )
+    ref = EclFile( ref_file )
+
+    for kw in ["TRANX" , "TRANY" , "TRANZ"]:
+        flow_kw = flow[kw][0]
+        ref_kw = ref[kw][0]
+
+        assert( flow_kw.equal_numeric( ref_kw , epsilon = 1e-3 ))
+
+        
+
+
+
+
+flow = sys.argv[1]
+data_file = sys.argv[2]
+ref_init_file = sys.argv[3]
+
+
+with TestAreaContext("flow_init") as ta:
+    ta.copy_file( data_file )
+    subprocess.check_call( [flow , data_file ] )
+
+    compare_files( os.path.splitext( os.path.basename( data_file ) )[0] + ".INIT" , ref_init_file )
+
+
```

Comment:
These are curious questions rather than review comments, I do not pretend to be competent enough in Python to review it (although the program reads quite well).
 - What does the TestAreaContext do (and its string argument)?
 - Why do we need to take [0] on lines 15, 16 and 34?
 - Does the variable flow on line 11 have anything to do with the variable on line 25 (hope not, just confusing)?